src/android/CameraLauncher.java

```diff
@@ -387,12 +394,16 @@ public void getImage(int srcType, int returnType) {
             } else {
                 intent.setAction(Intent.ACTION_GET_CONTENT);
                 intent.addCategory(Intent.CATEGORY_OPENABLE);
+                if (destType == FILE_URI) {
```

Comment:
To pick a bit, this should probably also check if mediaMaxCount is > 1, otherwise single image is expected, thus multi selection should not be used and same below for video multi-select